### DISCUSSION TITLE
Welcome to NodeX Discussions!

### DISCUSSION THREAD

**Discussion by nodexone** [nodesxploit/testnet#2] [Q&A] [answered]
Halo guys, kalo ada pertanyaan seputar testnet bisa kita diskusikan disini yaa

**Comment by 0xCryptoPunk** [comment, marked as answer]
bang, untuk deweb sendiri kapan testnet sirius mulai?

**Reply by nodexone** [reply]
sirius sudah mulai dari kemarin, cuman masih ada beberapa bug, jadi kemungkinan chain bakal halted lagi
